edb/pgsql/types.py

```diff
@@ -512,6 +512,9 @@ def _get_ptrref_storage_info(
 
     target = ptrref.out_target
 
+    if isinstance(ptrref, irast.TypeIntersectionPointerRef):
```

Comment:
Actually, could we handle this at the call site? Or else raise in this case unless `allow_missing` is set?

The typechecker isn't catching this because of overloads, but if `allow_missing` and `link_bias` are both `False`, this function isn't supposed to return `None`.

Reply:
That makes sense.